homeassistant/components/openhardwaremonitor/sensor.py

```diff
@@ -71,6 +71,11 @@ def name(self):
         """Return the name of the device."""
         return self._name
 
+    @property
+    def unique_id(self):
+        """Return a unique, Home Assistant friendly identifier for this entity."""
+        return self._name
```

Comment:
Names cannot be used as unique ID.

See: https://developers.home-assistant.io/docs/entity_registry_index/#unique-id-requirements

Closing this PR for that reason.

../Frenck